tests/test_insufficient_balances.py

```diff
@@ -0,0 +1,64 @@
+import itertools
+
+import brownie
+import pytest
+from brownie import ETH_ADDRESS
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup(mint_alice, approve_alice, mint_bob, approve_bob, set_fees):
+    set_fees(4000000, 5000000000, include_meta=True)
+
+
+def test_add_liquidity_insufficient_balance(
+    chain,
+    alice,
+    bob,
+    charlie,
+    swap,
+    n_coins,
+    wrapped_decimals,
+    underlying_decimals,
+    wrapped_coins,
+    underlying_coins,
+    initial_amounts,
+):
+    # attempt to deposit 10x more funds than user has
+    amounts = [i * 10 for i in initial_amounts]
```

Comment:
for this check, can you iterate and try depositing amounts that would work for every token, except `balanceOf(alice) + 1` for one token?